src/infrastructure/database/repositories/games.repository.js

```diff
@@ -54,18 +44,8 @@ export class GamesRepository {
   async getXgamesCheckedMoreThanYmsAgo(amount, ms) {
     return (
       await this.#dbClient
-        .get("history_checks")
+        .get("games")
         .aggregate([
-          {
```

Comment:
why is this changed? simplified?

Reply:
Yes, simplified/refactored. These line are unnecessary.